src/uartproxy.c

```diff
@@ -180,6 +196,9 @@ int uartproxy_run(struct uartproxy_msg_start *start)
 
         switch (request.type) {
             case REQ_NOP:
+                enabled_features = request.prequest.opcode & PROXY_FEAT_ALL;
```

Comment:
Can you add a `features` member to the union, to avoid overloading the proxy request struct?

Reply:
Makes sense, will fix.